test/functional/fixtures/package_plugin_config.json

```diff
@@ -2,6 +2,9 @@
   "global_args": {},
   "plugins": {
     "PackagePlugin": {
+      "collection_args": {
+        "rocm_regex": "rocm|hip|hsa|amdgpu"
```

Comment:
Did you mean this as analysis_args? as far as i can tell you have not added collection_args to this plugin